bin/git-create-branch

```diff
@@ -1,8 +1,35 @@
 #!/usr/bin/env bash
 
-branch=$1
-test -z $branch && echo "branch required." 1>&2 && exit 1
+for param in $*
+do
+	case $param in
+		-r|--remote)
+			if [ $# -eq 2 ]
+			then
+				BRANCH=$2
+			else			
+				REMOTE=$2
+				BRANCH=$3
```

Comment:
If we run `git-create-branch xx -r origin` :
The remote would be `-r`, while the branch would be `origin`.